### ISSUE TITLE
Dropdown Div Does Not React to Changes in Field/Block Size

### ISSUE THREAD

**Issue by BeksOmega** [google/blockly#2430]
### Problem statement

If the size of the block/field changes  the dropdown div can become 'disconnected' from the field.

![DropdownDivStatic](https://user-images.githubusercontent.com/25440652/57081642-04116600-6caa-11e9-8d20-2d9a5ef399b1.gif)

Though, the advantage of it not moving is that you don't have to keep chasing the div to toggle through options.

### Expected Behavior

Maybe the arrow should stretch to maintain it's connection to the field? Sort of like how icon bubbles work. This doesn't really help in the opposite case where the field grows though:
![DropdownDivGrow](https://user-images.githubusercontent.com/25440652/57082059-d7118300-6caa-11e9-9ec2-2b175e4afb24.gif)

I don't think the div should be repositioned each time, because that results in chasing behavior.

### Actual Behavior

See the above two gifs.

### Steps to Reproduce

If you want to play around with it you can see my changes [here](https://github.com/google/blockly/compare/develop...BeksOmega:demos/CustomFields).
The below xml will add a block with a turtle field to the workspace.
```
<xml xmlns="http://www.w3.org/1999/xhtml">
  <block type="turtle_basic"></block>
</xml>
```

### Operating System and Browser

I am assuming this is not browser specific.

### Additional Information

Not /necessarily/ that big of a deal that this happens, but I figured I would report it anyway.

**Comment by RoboErikG** [comment]
Yeah, I think you're right that moving it would be worse, so the only thing we could reasonably do is adjust the arrow.

**Comment by rachel-fenichel** [comment]
I agree with your analysis--moving the arrow would make sense when the dropdown is shrinking, but not when it's growing, and overall I'd call it low priority.

Also omg those turtles.
